### ISSUE TITLE
Trying to enable/disable 2fa fails with "security token" error

### ISSUE THREAD

**Issue by paladox** [matomo-org/matomo#13863]
Hi, we checked out "3.8.0-b4" for the new 2fa feature. (since we tried out the plugin but needed to disable it for a account which core had support for). But since upgrading to 3.8.0-b4 users cannot disable it from their account if they previously have done that or new users cannot enable it on their account.

When we get to the enter password to confirm prompt it fails afterwards with "Could not verify the security token on this form."

**Comment by paladox** [comment]
The problem seemed to happen when we installed the 2fa plugin. Then we upgraded to the 3.8 beta after noticing that it supported 2fa natively. (we needed to unlock someones account with the 2fa console command). This affected https://matomo.miraheze.org but seems to work now :).

We updated from 3.7 to 3.8 beta from doing git checkout <3.8 beta branch> (and then the db update through the ui), and the console command we ran to fix it was <code>./console twofactorauth:disable-2fa-for-user --login=user</code>
